### ISSUE TITLE
logger may log messages twice

### ISSUE THREAD

**Issue by schmir** [trustlines-network/relay#203]
```
~/trustlines/relay (git)-[wip/synchronized-relay-tx] % python
Python 3.6.6 (default, Sep 12 2018, 18:26:19) 
[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import relay.api.app
>>> import relay.api.resources
>>> relay.api.resources.logger.info("foo")
2018-09-25 23:04:51,016[INFO] api.resources: foo
2018-09-25 23:04:51,016[INFO] api.resources: foo
>>> 
```
Changing the get_logger call in `relay.api.resources` to use `apiresources` as name fixes it:
```
~/trustlines/relay (git)-[wip/synchronized-relay-tx] % python
Python 3.6.6 (default, Sep 12 2018, 18:26:19) 
[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import relay.api.app
>>> import relay.api.resources
>>> relay.api.resources.logger.info("foo")
2018-09-25 23:08:04,543[INFO] apiresources: foo
>>> 
```

**Comment by berndbohmeier** [comment]
should be fixed by 2fc3928

**Comment by schmir** [comment]
No, that above commit is only a workaround. We're still not able to choose our name for the logger freely without risking log messaged getting logged twice.
